netket/vqs/mc/mc_state/expect_grad.py

```diff
@@ -177,8 +195,8 @@ def grad_expect_operator_kernel(
     model_state: PyTree,
     σ: jnp.ndarray,
     local_value_args: PyTree,
+    **kwargs,
```

Comment:
```suggestion
    _return_estimators = False
```
Is there a particular reason for doing it like this? It makes checking quite complicated